packages/core/src/plugins/pinecone/PineconeVectorDatabase.ts

```diff
@@ -9,23 +8,36 @@ import {
   coerceType,
 } from '@ironclad/rivet-core';
 
+async function sha256(message: string): Promise<string> {
+  // encode as UTF-8
+  const msgBuffer = new TextEncoder().encode(message);
+
+  // hash the message
+  const hashBuffer = await crypto.subtle.digest('SHA-256', msgBuffer);
```

Comment:
My main concern is that this bumps up `rivet-node`'s requirement to be Node 19, because on node 16/17/18 it's behind the `--experimental-node-webcrypto`flag. Since I'd like rivet to integrate with _slightly_ older codebases, can we pull in `crypto-js` instead for a pure JS implementation? This will be necessary too for the Python integration, since PythonMonkey probably won't have a global `crypto` object [yet]

Reply:
Migrated to crypto-js.